frontend_build/src/webpack.config.base.js

```diff
@@ -62,9 +77,9 @@ var config = {
   ],
   resolve: {
     alias: {
-      'kolibri_module': path.join(__dirname, '..', 'kolibri', 'core', 'assets', 'src', 'kolibri_module'),
-      'core-base': path.join(__dirname, '..', 'kolibri', 'core', 'assets', 'src', 'core-base'),
-      'core-theme.styl': path.join(__dirname, '..', 'kolibri', 'core', 'assets', 'src', 'core-theme.styl'),
+      'kolibri_module': path.join(__dirname, '..', '..', 'kolibri', 'core', 'assets', 'src', 'kolibri_module'),
```

Comment:
This should really be `path.resolve`, because it's cleaner, but I am fixing this up in my WIP API PR, so will just update there.